src/actors/common/cache.rs

```diff
@@ -125,6 +126,19 @@ impl<T: CacheItemRequest> Cacher<T> {
                     "hit" => "true"
                 );
 
+                configure_scope(|scope| {
+                    scope.set_extra(
+                        &format!("cache.{}.cache_path", name),
+                        format!("{:?}", path).into(),
+                    );
+                });
+
+                log::debug!(
```

Comment:
_nit_: Since this happens *very* frequently and we have a tag for this, can we make this `log::trace!`?